src/picojson.h

```diff
@@ -210,7 +210,7 @@ namespace picojson {
     if (
 #ifdef _MSC_VER
         ! _finite(n)
-#elif __cplusplus>=201103L || !(defined(isnan) && defined(isinf))
+#elif (__cplusplus>=201103L && !defined(__AROS__)) || !(defined(isnan) && defined(isinf))
 		std::isnan(n) || std::isinf(n)
```

Comment:
this one makes we wonder why non-emscripten builds even have picojson included

Reply:
...because `CreateRequestMapping` is not guarded and generally the async stuff is so ugly nobody wants to touch it ;)

Reply:
okay, I will touch it later and then undo that one-line change ;)